doc/Docker.md

```diff
@@ -0,0 +1,51 @@
+Docker
+=====
+
+* [Home](help)
+
+
+Auto-Install
+---
+
+With the autoinstall function of friendice, you can automatically create docker environments, where some of the important configs (f.e the db credentials) are saved inside your volume beside your friendica data.  
```

Comment:
- `Friendica` instead of `friendice` or `friendica`
- `Docker` instead of `docker`